pangea-sdk/service/vault/service.go

```diff
@@ -30,12 +30,12 @@ type Client interface {
 	JWTVerify(ctx context.Context, input *JWTVerifyRequest) (*pangea.PangeaResponse[JWTVerifyResult], error)
 }
 
-type vault struct {
+type Vault struct {
 	*pangea.Client
 }
 
 func New(cfg *pangea.Config) Client {
-	cli := &vault{
+	cli := &Vault{
```

Comment:
It won't be a breaking change due to we still return a `Client` but IMO we should keep it private. We can talk it with @ggallien13 @govind-salinas-pangea also maybe